src/psyclone/line_length.py

```diff
@@ -13,9 +13,11 @@
 
 def find_break_point(line, max_index, key_list):
     ''' find the most appropriate break point for a fortran line '''
-
```

Comment:
Unfortunately (for you) this is ancient code. Please could you change the copyright header to be the one we now use and document the arguments and return value/type of this method.
Please could you also remove the `from __future__` import near the top.

Reply:
Please could you also move the `import re` at L53 to the top (pylint complains).

Reply:
Updated.
There is no `from __future__` import in my version of the file I can find.

Reply:
Ah future is in the tests - i removed it there.

Reply:
Oops, my mistake and thanks.